src/test/java/org/walkmod/javalang/compiler/SymbolVisitorAdapterTest.java

```diff
@@ -2162,4 +2162,49 @@ public void testErrorWithStaticAndNotStaticImports() throws Exception{
 	   
 	   Assert.assertNotNull(cu);
 	}
+
+	private final String genericColumnAccessor = ""
+			+ "package apkg;\n"
+			+ " public class Column<T> {\n"
+			+ "  public static final Column<Long> Maxtime = new Column<Long>();\n"
+			+ " }\n";
+
+	@Test
+	public void testTypeOfGenericConstant() throws Exception {
+		CompilationUnit cu = run(genericColumnAccessor);
+
+		Assert.assertTrue(true);
+		//SymbolVisitorAdapterAssertions.assertTypeOfGenericConstant(cu);
```

Comment:
These are the fluent assertions i wrote about via email. Useful but work in progress. Details in email.